PI Vision 2019 Extensibility Guide.md

```diff
@@ -1,27 +1,5 @@
 # PI Vision 2019 Extensibility Guide
 
-OSIsoft, LLC  
-1600 Alvarado St.  
-San Leandro, CA 94577 USA  
-Tel: (1) 510-297-5800  
-Fax: (1) 510-357-8136  
-Web: http://www.osisoft.com
-
-PI Vision 2019 Extensibility Guide
-
-© 2016 - 2019 by OSIsoft, LLC. All rights reserved.
-No part of this publication may be reproduced, stored in a retrieval system, or transmitted, in any form or by any means, mechanical, photocopying, recording, or otherwise, without the prior written permission of OSIsoft, LLC.
-
-OSIsoft, the OSIsoft logo and logotype, Managed PI, OSIsoft Advanced Services, OSIsoft Cloud Services, OSIsoft Connected Services, PI ACE, PI Advanced Computing Engine, PI AF SDK, PI API, PI Asset Framework, PI Audit Viewer, PI Builder, PI Cloud Connect, PI Connectors, PI Data Archive, PI DataLink, PI DataLink Server, PI Developer’s Club, PI Integrator for Business Analytics, PI Interfaces, PI JDBC driver, PI Manual Logger, PI Notifications, PI ODBC, PI OLEDB Enterprise, PI OLEDB Provider, PI OPC HDA Server, PI ProcessBook, PI SDK, PI Server, PI Square, PI System, PI System Access, PI Vision, PI Visualization Suite, PI Web API, PI WebParts, PI Web Services, RLINK, and RtReports are all trademarks of OSIsoft, LLC. All other trademarks or trade names used herein are the property of their respective owners.
-
-U.S. GOVERNMENT RIGHTS
-
-Use, duplication or disclosure by the U.S. Government is subject to restrictions set forth in the OSIsoft, LLC license agreement and as provided in DFARS 227.7202, DFARS 252.227-7013, FAR 12.212, FAR 52.227, as applicable. OSIsoft, LLC.
-
-Version: 3.4
-
-Published: 4 February 2020
-

```

Comment:
Removed all legal details to match what we have in the other GitHub markdown files. Can add back if Legal determines we need it